client/stylesheets/_modal.scss

```diff
@@ -902,6 +902,19 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 	height: 100px;
 }
 
+.modFeedbackInitial button[data-target="#feedbackBuyerUnsuccessfulModal"], .modFeedbackInitial button[data-target="#feedbackBuyerNeverModal"] {
+	padding: 11px 50px !important;
```

Comment:
!important should not be used<br>Line should be indented with spaces, not tabs<br>Properties should be ordered background-color, border, border-radius, color, padding